src/Microsoft.Performance.SDK/Processing/ColumnMetadata.cs

```diff
@@ -160,7 +160,7 @@ public ColumnMetadata(ColumnMetadata other)
         ///     on the data in the first row. Use <see cref="NameProjection"/>
         ///     for dynamic names.
         /// </summary>
-        public string Name { get; }
+        public string Name { get; init; }
```

Comment:
>public string Name { get; init; } [](http://example.com/codeflow?start=8&length=33)

do we want a nullorwhitespace check on this init like we have in the constructors?

Reply:
We can do that, but not without declaring our own backing property :(